package.json

```diff
@@ -21,7 +21,7 @@
     "lint": "eslint packages tests --cache --fix --ext=jsx --ext=js",
     "pretest": "npm run lint --silent",
     "lint:ci": "eslint packages tests --ext=jsx --ext=js",
-    "test": "jest --runInBand tests/*",
+    "test": "jest --runInBand",
```

Comment:
Why did you change that? It looks like now there is docs tests included in the test that fail

Reply:
I removed it because I wanted to run a single file of test with `npm run test -- file-name` and couldn't do that with the previous setup. all tests seem to be working when running it by `npm run test` but I'll replace it if you have encountered an issue.